scripts/cci-test-timings.data.json

```diff
@@ -0,0 +1,1074 @@
+{
+  "lastUpdated": "2022-12-31T02:03:46.162Z",
+  "totalTestFiles": 266,
```

Comment:
Is this the same as the length of `timingData`? If so, I would think computing the length is better to avoid having to ensure both are in sync.

Reply:
this is just some metadata to provide at a glance; its helpful to quickly see when the last updated time was & how many files we having timing data for